pkgs/tools/networking/termscp/default.nix

```diff
@@ -5,23 +5,24 @@
 , openssl
 , pkg-config
 , rustPlatform
+, Cocoa
 , Foundation
 , Security
 , stdenv
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "termscp";
-  version = "0.8.2";
+  version = "0.9.0";
 
   src = fetchFromGitHub {
     owner = "veeso";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-7T3VmcI9CWrKROQ0U2du2d8e0A6XnOxpd8Zl0T4w+KQ=";
+    sha256 = "sha256-iazp3Qx2AivuL+S1Ma/64BLJtE46tc33dq5qsgw+a6Q=";
   };
 
-  cargoSha256 = "sha256-WuoN7b9Fw2Op8tck4ek8gyufInlbPkDHHtLAsbG1NLE=";
+  cargoSha256 = "sha256-FBW3Hl67Efnc/sNGM1LQw6msWHCYRj3KwfmSD2lpbUc=";
 
```

Comment:
```suggestion

  NIX_CFLAGS_COMPILE = lib.optionals stdenv.isDarwin [
    "-framework" "AppKit"
  ];

```

Reply:
This feels like it should happen automatically. I'm as confused as people in https://github.com/NixOS/nixpkgs/issues/160876#issuecomment-1060113970